app/tests/lib/helpers.js

```diff
@@ -68,6 +68,7 @@ define(function (require, exports, module) {
       fn();
     } catch (e) {
       done(e);
+      return;
```

Comment:
I'm bike-shedding to the extreme here, so feel free to ignore, but I notice this could also be achieved without the need for an explicit `return` by moving the outer `done()` in to the `try` block. Not sure why but I find the arrangement here a little bit jarring to read.

Reply:
I imagine it's jarring because you have to think about the flow control this way and forced thought is annoying. Will update.